duels-plugin/src/main/resources/lang.yml

```
@@ -445,6 +482,8 @@ COMMAND:
       - '{LINE}'
 
   sub-command-usage: '{PREFIX} &f/%command% %usage% &e- &7%description%'
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)